### PR TITLE
Add performance documentation

### PR DESCRIPTION
**Pull request by joeldenning**

### CODE REVIEW

**Comment by joeldenning** [comment]
@guybedford this is ready for your review. I removed the depcache section since that hasn't landed yet in SystemJS and also isn't really available for native import maps

**Comment by joeldenning** [comment]
@guybedford any update on this?

File: README.md [hunk 1]
```diff
@@ -60,6 +60,16 @@ The following extras are included in system.js loader by default, and can be add
 
 Since all loader features are hookable, custom extensions can be easily made following the same approach as the bundled extras. See the [hooks documentation](docs/hooks.md) for more information.
 
+## Performance
+
+SystemJS can load modules in less than a millisecond, and its performance is very nearly identical to that of native ES modules. The following performance benchmark was run by loading 450 javascript modules (all of `@babel/core`) on a Macbook pro with fast wifi internet connection. Each test was the average of five page loads in Chrome 80.
```
Comment (guybedford) [inline README.md]:
```suggestion
SystemJS can load multiple modules in less than a millisecond with its performance close to around a factor of 1.5 times the performance of native ES modules. The following performance benchmark was run by loading 450 javascript modules (all of `@babel/core`) on a Macbook pro with fast wifi internet connection. Each test was the average of five page loads in Chrome 80.
```

**Comment by LarsDenBakker** [comment]
This is very useful information, thanks for putting this up. Interesting to see the comparison to `es-module-shims`.

**Comment by joeldenning** [comment]
Updated with your feedback

**Comment by dcleao** [comment]
> with its performance around a factor of 1.5 times the performance of native ES modules

Is this equivalent to saying that "SystemJS modules take around twice the time to load of a corresponding native ES module"?

**Comment by guybedford** [comment]
No it's 50% longer, not 100% longer.

**Comment by dcleao** [comment]
Ok, sorry... I was just rounding things up too much (e.g. 81ms / 49ms = 1.65).

**Pull request merged**